flow/client_server/templates/navbar/base.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
iirc It's part of the standard, if you don't do it most browsers are smart enough to pretend it's there.